stm/stm.mli

```diff
@@ -51,6 +51,9 @@ val edit_at : Stateid.t -> [ `NewTip | `Focus of focus ]
 (* Evaluates the tip of the current branch *)
 val finish : unit -> unit
 
+(* For testing only, do not use *)
```

Comment:
Ditto, please refine to `Hack for fake_ide. Do not use`.

Reply:
Sorry but I don't see how writing that it is a hack is an improvement. "Do not use" look enough to me.

Reply:
I'd like "testing" to become more concrete.

Reply:
testing may have many meanings in this context IMO.

Reply:
Ok, I can write "for internal use only, do not use".

Reply:
well, if you mention `fake_ide` the person looking next won't have to guess why is this leftover there I think.